src/users_router.rs

```diff
@@ -344,6 +344,7 @@ describe! setup_tests {
                 assert!(false);
             }
         };
+        remove_test_db();
```

Comment:
Would I be really picky if i ask to spend some times knowing why it's happening? Cause it looks a bit weird to me right now with all those calls.